pkgs/applications/version-management/gitolite/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ nettools perl ];
   nativeBuildInputs = [ makeWrapper ];
-  propagatedBuildInputs = [ git ];
```

Comment:
Why did you add coreutils to propagatedBuildInputs? coreutils is per default available in build environments.
I have to admit, I do understand why git is propagated.

Reply:
It might be superfluous to have added it here. But the intention was to ensure it is added to the runtime environment. This is probably not the right thing to do here. I’ll fix.